glean-core/android/src/main/java/mozilla/telemetry/glean/private/PingType.kt

```diff
@@ -17,14 +17,16 @@ import mozilla.telemetry.glean.rust.toByte
  */
 class PingType(
     internal val name: String,
-    includeClientId: Boolean
+    includeClientId: Boolean,
+    sendIfEmpty: Boolean = false
```

Comment:
Why are we providing a default value here? Isn't `glean_parser` taking care of adding it?

Reply:
Considered it mostly for backwards-compatibility.
Even for files that are not regenerated by the parser they keep working.

Reply:
Thanks for explaining! Would you consider changing this so that any problem will surface? We don't expect files to be manually crafted except inside our core.

Reply:
Sure, I can. guess I might need to land the parser changes and update it here first to make it all work though

Reply:
Sounds good to me!